app/controllers/application_controller.rb

```diff
@@ -40,4 +41,15 @@ def logo_class
     "cf-logo-image-default"
   end
   helper_method :logo_class
+
+  def set_raven_user
+    if current_user && ENV["SENTRY_DSN"]
+      # Raven sends error info to Sentry.
+      Raven.user_context(
+        css_id: current_user.css_id,
```

Comment:
I would also duplicate the `css_id` field and call it `id`. Apparently, sentry expects the id to actually be called `id`. Otherwise, it falls back to IP address, which is far less useful.

https://docs.sentry.io/learn/context/#capturing-the-user